src/SolverInterface/sets.jl

```diff
@@ -60,7 +60,34 @@ struct SecondOrderCone <: AbstractSet
 end
 
 #ExponentialCone
-#PositiveSemidefiniteCone
+
+"""
+    PositiveSemidefiniteCone{Scaled}(n)
```

Comment:
What would be the names then :-P UnscaledPositiveSemidefiniteCone and ScaledPositiveSemidefiniteCone ?